license_header.txt

```diff
@@ -0,0 +1,15 @@
+
+
+Copyright Red Hat
```

Comment:
One other small question, I noticed we are removing the years, curious as to why..

Reply:
During devtools week discussions (Sep 11-15) we decided to make this change to follow Red Hat legal's guidelines on copyright labeling under the license headers.